gomega_dsl.go

```diff
@@ -52,12 +52,12 @@ func RegisterFailHandler(handler types.GomegaFailHandler) {
 //
 //You'll need to call this at the top of each XUnit style test:
 //
-// func TestFarmHasCow(t *testing.T) {
-//     RegisterTestingT(t)
+//    func TestFarmHasCow(t *testing.T) {
```

Comment:
There was a bit of inconsistency in code indentation. The prevailing pattern (~100 instances vs ~30 instances) was to use four space characters.

# Prod
<img width="200" alt="screen shot 2018-06-12 at 5 46 27 pm" src="https://user-images.githubusercontent.com/1851748/41321407-69b37bdc-6e69-11e8-96d8-50b22d5e5a81.png">

# Local
<img width="200" alt="screen shot 2018-06-12 at 5 46 40 pm" src="https://user-images.githubusercontent.com/1851748/41321408-69c0e344-6e69-11e8-9011-b2fabdf269be.png">

Reply:
This was a case where only one space was used, so it seems like the code indentation wasn't working at all in prod